.github/workflows/release.yaml

```
@@ -30,7 +30,7 @@ jobs:
         with:
           version: pnpm changeset version
           publish: pnpm changeset publish
-          commit: "chore(release): Version Packages"
+          commit: "chore(release): version-packages"
```

```suggestion
          commit: "chore(release): version packages"
```
첫 글자가 대문자만 아니면 됐던 것 겉아!

---

@noahluftyang 

하이폰 지웠습니당.